web/client/epics/dashboard.js

```diff
@@ -108,7 +108,7 @@ export const handleDashboardWidgetsFilterPanel = (action$, {getState = () => {}}
                 .merge(action$.ofType(TOGGLE_CONTROL).filter(({control, property} = {}) => control === "widgetBuilder" && (!property === false))))
             .concat(
                 Rx.Observable.of(// drawSupportReset(),
-                    setControlProperty('queryPanel', "enabled", false)
+                    setControlProperty('queryPanelWithMap', "enabled", false)
```

Comment:
I think you can keep the originals

Reply:
this would mean to use one single plugin